wrapper/ibus/setup/main.py

```diff
@@ -89,17 +89,14 @@ def __init__(self, name, default):
 
     def read(self):
         section, key = self.__get_config_name()
-        return self.config.get_value(section, key, self.default)
+        res =  self.config.get_value(section, key)
+        if res is None:
+            return self.default
+        return self.__unwrappers[type(self.default)](res)
 
     def write(self, v):
         section, key = self.__get_config_name()
-        return self.config.set_value(section, key, self.wrap(v))
-
-    def wrap(self, v):
```

Comment:
actually, i like this little helper apart from its name -- would be better if it was started with `__`. as it's "private".